packages/suite/src/components/wallet/WalletLayout/AccountBanners/TaprootBanner.tsx

```diff
@@ -81,14 +70,13 @@ export const TaprootBanner = ({ onClose }: TaprootBannerProps) => {
             </Point>
             <Point>
                 <StyledIcon icon="CHECK" size={20} color={theme.TYPE_GREEN} />
```

Comment:
```jsx
color={theme.iconPrimaryDefault}
```

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/11099/commits/6ee47baa47af021ca155ca51626de835864a58f0